jobs/e2e_node/containerd/k8s-release-branches/image-config-1.25.yaml

```diff
@@ -0,0 +1,5 @@
+images:
+  cos-101:
+    image_family: cos-101-lts
```

Comment:
is 1.25 and 1.26 use the same version?

Reply:
Yes, but I think it's better to leave separate configs: potentially we want to add more images, and these might differ between branches.